server/src/services/nrm.service.es6

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Is this file used anywhere, or is it presumptive about what we'll need when the NRM service is actually implemented? Just curious because I don't see it referenced anywhere. :)